runtime/src/main/java/io/quarkiverse/semantickernel/SemanticKernelProducer.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkiverse.semantickernel;
+
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Inject;
+
+import com.azure.ai.openai.OpenAIAsyncClient;
+import com.microsoft.semantickernel.Kernel;
+import com.microsoft.semantickernel.SKBuilders;
+
+public class SemanticKernelProducer {
+    // Just a >placeholder<, before the actual implementation, to enable the
```

Comment:
What shall we do with this comment then ?

Reply:
I think we should keep the SemanticKernelProducer as is, for now, just for the purpose to run the semantic function tests, and open a new issue to work on it properly

Reply:
Pk @morphy76 . Could you create the issue and add it to this comment so we keep track?

Reply:
Issue #5